Detections/CommonSecurityLog/Zscaler-LowVolumeDomainRequests.yaml

```diff
@@ -19,25 +19,31 @@ relevantTechniques:
 query: |
 
   let timeRange = 1d;
+  let scriptExtensions = dynamic([".php", ".aspx", ".asp", ".cfml"]);
   //The number of URI's seen to be suspicious, higher = less likely to be suspicious
   let uriThreshold = 1;
-  let suspiciousURLs =
   CommonSecurityLog
   | where TimeGenerated >= ago(timeRange)
-  // Only look at connections that were allowed through the web proxy. Comment line out for all events including those which were blocked
+  // Only look at connections that were allowed through the web proxy
   | where DeviceVendor =~ "Zscaler" and DeviceAction =~ "Allowed"
-  // Only look where some data was exchanged. Comment out to see 0 byte connections
+  // Only look where some data was exchanged.
   | where SentBytes > 0 and ReceivedBytes > 0
   // Extract the Domain
   | extend Domain = iff(countof(DestinationHostName,'.') >= 2, strcat(split(DestinationHostName,'.')[-2], '.',split(DestinationHostName,'.')[-1]), DestinationHostName)
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), makelist(RequestURL), makelist(DestinationIP), makelist(SourceIP), numOfConnections = count() by Domain
+  | extend GetData=iff(RequestURL in "?", 1, 0)
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), makelist(RequestURL), makelist(DestinationIP), makelist(SourceIP), numOfConnections = count(), make_set(RequestMethod), max(GetData), max(RequestContext) by Domain
   // Determine the number of URIs that have been visited for the domain
   | extend destinationURI = arraylength(list_RequestURL)
   | where destinationURI <= uriThreshold
-  | where tostring(list_RequestURL) has_any (".php",".aspx")
-  ;
-  suspiciousURLs
+  | where tostring(list_RequestURL) has_any(scriptExtensions)
+  //Remove matches with referer
+  | where max_RequestContext == ""
+  //Keep requests where data was trasferred either in a GET with parameters or a POST
+  | where set_RequestMethod in~ "POST" or max_GetData == 1
```

Comment:
Same here - this needs parenthesis - set_RequestMethod in~ **("POST")**.  Again, you probably should be using "has" here as this is a set.